.github/workflows/main.yml

```diff
@@ -68,7 +68,7 @@ jobs:
       - name: Install Hive Module
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
-          ./mvnw install ${MAVEN_FAST_INSTALL} -pl presto-hive-hadoop2 -am
+          ./mvnw install ${MAVEN_FAST_INSTALL} -am -pl presto-hive-hadoop2
```

Comment:
it's more reasonable to have `-am` after `-pl`: "make this module and also dependencies" rather than "make also dependencies, make this module"

Reply:
I agree, which is why it was that way before, but having the module name at the end makes it easier to see, since that's the variable part, especially with the change below.